src/main/java/com/psychology/product/util/ResponseHandler.java

```diff
@@ -0,0 +1,25 @@
+package com.psychology.product.util;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.util.Map;
+
+public class ResponseHandler {
+    public static ResponseEntity<Object> generateResponse(String messege, HttpStatus status) {
+        Map<String, Object> map = Map.of(
```

Comment:
Be better to clearly indicate the type

Reply:
We use it in all project. If we add specific type need rewrite this method. But we try to don't repeat youself